troposphere/static/js/utilities/subscribe.js

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+
+import AllocationStore from "stores/AllocationStore";
+import BadgeStore from "stores/BadgeStore";
+import ClientCredentialStore from "stores/ClientCredentialStore";
+import ExternalLinkStore from "stores/ExternalLinkStore";
+import HelpLinkStore from "stores/HelpLinkStore";
+import ImageStore from "stores/ImageStore";
+import ImageVersionStore from "stores/ImageVersionStore";
+import ImageVersionMembershipStore from "stores/ImageVersionMembershipStore";
+import ImageVersionLicenseStore from "stores/ImageVersionLicenseStore";
+import ImageVersionScriptStore from "stores/ImageVersionScriptStore";
+import IdentityStore from "stores/IdentityStore";
+import ImageBookmarkStore from "stores/ImageBookmarkStore";
+import InstanceHistoryStore from "stores/InstanceHistoryStore";
+import ImageRequestStore from "stores/ImageRequestStore";
+import InstanceStore from "stores/InstanceStore";
+import InstanceTagStore from "stores/InstanceTagStore";
+import LicenseStore from "stores/LicenseStore";
+import ScriptStore from "stores/ScriptStore";
+import MaintenanceMessageStore from "stores/MaintenanceMessageStore";
+import MyBadgeStore from "stores/MyBadgeStore";
+import MembershipStore from "stores/MembershipStore";
+import ProfileStore from "stores/ProfileStore";
+import ProjectStore from "stores/ProjectStore";
+import ProjectExternalLinkStore from "stores/ProjectExternalLinkStore";
+import ProjectImageStore from "stores/ProjectImageStore";
+import ProjectInstanceStore from "stores/ProjectInstanceStore";
+import ProjectVolumeStore from "stores/ProjectVolumeStore";
+import ProviderMachineStore from "stores/ProviderMachineStore";
+import ProviderStore from "stores/ProviderStore";
+import ResourceRequestStore from "stores/ResourceRequestStore";
+import IdentityMembershipStore from "stores/IdentityMembershipStore";
+import StatusStore from "stores/StatusStore";
+import SSHKeyStore from "stores/SSHKeyStore";
+import QuotaStore from "stores/QuotaStore";
+import SizeStore from "stores/SizeStore";
+import TagStore from "stores/TagStore";
+import UserStore from "stores/UserStore";
+import VersionStore from "stores/VersionStore";
+import VolumeStore from "stores/VolumeStore";
+import AllocationSourceStore from "stores/AllocationSourceStore";
+
```

Comment:
We do until we remove the dynamic library setup. Take a look at the code comment below and let me know if it makes sense. Using strings is not ideal, not sure of a good replacement.